src/pages/Scoreboard.js

```diff
@@ -1,19 +1,68 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  Typography,
+} from "@material-ui/core";
+import { SCOREBOARD_HEADERS } from "../constants/constants";
 import { scoreboardTestData } from "../assets/data/scoreboardTestData";
 import { ScoreboardSongCard } from "../components/ScoreboardSongCard";
-import { SCOREBOARD_HEADERS } from "../constants/constants";
-import "../styling/scoreboard.css";
+
+const useStyles = makeStyles({
+  tableContainer: {
+    maxWidth: 1000,
+    margin: "auto",
+    marginTop: 50,
+  },
+  boldHeader: {
+    fontWeight: "bold",
+  },
+});
 
 export const Scoreboard = () => {
+  const classes = useStyles();
```

Comment:
Eventually we'll need to move towards a global theme and move it to a parent layer